### PR TITLE
First branch first real test

### PR DESCRIPTION
**Pull request by ethanstrominger**

### CODE REVIEW

File: src/credentials.xyz [hunk 1]
```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"346860590561-gp3lf3l1hp6c3hf4ld9kdlffr3m1kf16.apps.googleusercontent.com","project_id":"quickstart-1553108375563","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"7IotHlXkM6s6kW7ZNIbqFmFr","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```
Comment (eric-s-s) [inline src/credentials.xyz]: what is this file and should it be here? I seem to remember we created it for some reason and now totally forget why.

File: src/helloWorldRun.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import {sayHello} from './xample_hello_world';
```
Comment (eric-s-s) [inline src/helloWorldRun.js]: was this file here just to show import statements working?  if so, can we delete it?

File: src/smartCalendarUtils.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        name: vevent.getFirstPropertyValue("summary"),
+        starttime: moment(vevent.getFirstPropertyValue("dtstart")).format(),
+        endtime: moment(vevent.getFirstPropertyValue("dtend")).format(),
+        description: vevent.getFirstPropertyValue("description")
+    };
+    return element;
+}
+;
+
+export {getICalEvents};
```
Comment (eric-s-s) [inline src/smartCalendarUtils.js]: add newline.

File: src/test/helloWorld.test.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import {sayHello} from '../xample_hello_world';
```
Comment (eric-s-s) [inline src/test/helloWorld.test.js]: same as other file with `sayHello`. do these files still serve a purpose?  i think the `smartCalendarUtils.js` does what we want and demonstrates everything here.  i vote delete this file.

File: src/xample_hello_world.js [hunk 5]
```diff
@@ -1,12 +1,13 @@
 // const abc = require('./hello-world-functions');
```
Comment (eric-s-s) [inline src/xample_hello_world.js]: can we delete this file too?  happy to leave it, i just don't see a reason to keep it.

File: src/xample_list_events3.js [hunk 6]
```diff
@@ -0,0 +1,125 @@
+
```
Comment (eric-s-s) [inline src/xample_list_events3.js]: is this just the original example?

File: src/credentials.xyz (hunk 1 above)
Comment (ethanstrominger) [reply src/credentials.xyz]: This file is deleted now.

File: src/helloWorldRun.js (hunk 2 above)
Comment (ethanstrominger) [reply src/helloWorldRun.js]: This file is deleted now.

File: src/smartCalendarUtils.js (hunk 3 above)
Comment (ethanstrominger) [reply src/smartCalendarUtils.js]: I will do on next commit.

File: src/xample_hello_world.js (hunk 5 above)
Comment (ethanstrominger) [reply src/xample_hello_world.js]: I deleted it.

File: src/xample_list_events3.js (hunk 6 above)
Comment (ethanstrominger) [reply src/xample_list_events3.js]: Yes, modified to use async.

File: src/credentials.xyz (hunk 1 above)
Comment (ethanstrominger) [reply src/credentials.xyz]: Removed this file.

**Comment by boston-coding-collaborative** [comment]
All suggestions incorporated in other branch

**Pull request closed without merging**